camera.py

```diff
@@ -0,0 +1,56 @@
+class Camera:
```

Comment:
Import this in `main.py`.

Also add spacing between the functions for better readability.